src/gui/CMakeLists.txt

```diff
@@ -161,6 +161,8 @@ SET(QGIS_GUI_SRCS
   editorwidgets/qgsvaluerelationsearchwidgetwrapper.cpp
   editorwidgets/qgsvaluerelationwidgetfactory.cpp
 
+  gps/qgsgpsmarker.cpp
```

Comment:
If you're adding a new directory here, you'll also need to update:

https://github.com/qgis/QGIS/blob/master/doc/CMakeLists.txt#L102
https://github.com/qgis/QGIS/blob/master/python/CMakeLists.txt#L138

And then go through the:

scripts/sip_include.sh
scripts/sipify_all.sh

process